tests/foreman/ui/test_contenthost.py

```diff
@@ -141,6 +141,24 @@ def run_remote_command_on_content_host(command, vm_module_streams):
     assert result.return_code == 0
 
 
+@tier3
+def test_positive_rhel7_syspurpose(session, vm):
```

Comment:
From looking as Bugzilla https://bugzilla.redhat.com/show_bug.cgi?id=1703227, this is not specific only to RHEL7, It can be associated with RHEL8 as well. Can you please add coverage for that as well? Correct me if I'm wrong

Reply:
The bug is simply for a system that doesn't have syspurpose attributes. RHEL 7 is guaranteed not to and fits the case. Therefore, there is no need to use a RHEL 8 system as well.